Flutter/calendar/appointments.md

```diff
@@ -130,6 +130,86 @@ events.notifyListeners(CalendarDataSourceAction.reset, null);
 {% endhighlight %}
 {% endtabs %}
 
+## Notify listener
+
+By call this method whenever the datasource changes, to notify any clients the datasource may have. We have provided support to perform three operations like Add, Remove and Reset. NotifyListener is used to notify the dataSource may have changed.
+
+#### Add Action
+
+We need to notify the datasource changes due to add the appointment from the collection. 
```

Comment:
Check the content in quillbot and make content appropriate meaning

Reply:
I have checked the content in quillbot and updated the content in my latest commit.